### PR TITLE
[RFR] Added source to npm

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

**Comment by Kmaschta** [comment]
Why add sources to npm ?

And there is no comma on the last element of a list with JSON :cactus:

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "files": [
     "*.md",
     "docs",
-    "lib"
+    "lib",
+    "src",
```
Comment (fzaninotto) [inline package.json]: not JSON

Comment (djhi) [inline package.json]: @Kmaschta in order to be able to use the lib with a package reference to the repository

**Comment by djhi** [comment]
@Kmaschta in order to be able to use the lib with a package reference to the repository

**Comment by djhi** [comment]
Otherwise if you do:

`npm i --save marmelab/admin-on-rest#the_branch_i_want_to_test_real_bad`

You only receive the docs

**Comment by jpetitcolas** [comment]
@djhi: Not sure that's a good idea. Using `npm link` should fix your issue. 

``` sh
cd admin-on-rest
git checkout the_branch_i_want_to_test_real_bad
npm link

cd ../my-project
npm link admin-on-rest
```
And you can test your branch on your real project.

Embedding `src` folder would add useless files for end users IMO.

**Comment by fzaninotto** [comment]
:+1: @djhi could you test that and revert your PR if it makes sense?

**Pull request merged**